service-users-specs/src/test/java/io/pestakit/users/api/spec/steps/RetrivalDefinition.java

```diff
@@ -0,0 +1,68 @@
+//package io.pestakit.users.api.spec.steps;
```

Comment:
If this file is not used, please git remove it instead of keeping it commented.

Reply:
68d4a91c212b62d6df809f1f2cf8202a8cf5bbb2